### DISCUSSION TITLE
Gruntwork Pipelines: use OpenTofu

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#808] [How do I] [answered]
Hello, as a Gruntwork Pipelines user, how can I use the OpenTofu binary and control the version? 

---

<ins datetime="2024-03-15T15:59:46Z">
  <p><a href="https://support.gruntwork.io/hc/requests/110826">Tracked in ticket #110826</a></p>
</ins>

**Comment by ZachGoldberg** [comment]
FYI - There's an internal ticket tracking this work here - https://linear.app/gruntwork/issue/DEV-114/support-opentofu

**Reply by ZachGoldberg** [reply, marked as answer]
pipelines-execute v3.0.0-beta1 includes a new flag `TF_BINARY` which can be set to either `terraform` or `opentofu` (in addition to the existing `TF_VERSION` parameter).  By default, starting in v3.0.0 we'll use OpenTofu unless the binary is forcibly set to terraform by the calling workflow.
